index.src.html

```diff
@@ -1196,197 +1171,9 @@ <h4 algorithm id="passwordcredential-matching">
-          domain</a>, then skip the remaining substeps.
-
-      2.  Let |data| be a copy of |credential|'s
-          {{PasswordCredential/additionalData}} attribute if it is not
-          `undefined`, and a new {{FormData}} object otherwise.
-
-      3.  Let |list| be an empty list.
-
-      4.  If |data| is a {{FormData}} object, then:
-
-          1.  Set |list| to a copy of |data|'s <a for="FormData">entries</a>.
-
-          2.  Remove all <a>entry</a> objects from |list| whose
-              <a for="entry">name</a> is either |credential|'s
-              {{PasswordCredential/idName}} or |credential|'s
-              {{PasswordCredential/passwordName}}.
-
-          3.  Let |entry| be a new <a>entry</a> whose <a for="entry">name</a>
-              is |credential|'s {{PasswordCredential/idName}} and whose
-              <a for="entry">value</a> is |credential|'s {{Credential/id}}.
-
-          4.  Append |entry| to |list|.
-
-          5.  Let |entry| be a new <a>entry</a> whose <a for="entry">name</a>
-              is |credential|'s {{PasswordCredential/passwordName}} and whose
-              <a for="entry">value</a> is |credential|'s {{[[password]]}}.
-
-          6.  Append |entry| to |list|.
-
-          7.  Push the result of running the <a>`multipart/form-data` encoding
-              algorithm</a>, with |list| as the form data set and with "`utf-8`" as
-              the explicit character encoding, to |body|'s
-              <a for="body">stream</a>.
-
-          8.  Set |Content-Type| to `multipart/form-data;boundary=`, followed by the
-              <a>`multipart/form-data` boundary string</a> generated by the
-              multipart/form-data encoding algorithm.
-
-          Otherwise |data| is a {{URLSearchParams}} object, so:
-
-          1.  Set |list| to a copy of |data|'s
-              <a for="URLSearchParams">list</a>.
-
-          2.  Remove all name-value pairs from |list| whose name is either
-              |credential|'s {{PasswordCredential/idName}} or |credential|'s
-              {{PasswordCredential/passwordName}}.
-
-          3.  Append a new name-value pair to |list| whose name is 
-              |credential|'s {{PasswordCredential/idName}} and whose value is
-              |credential|'s {{Credential/id}}.
-
-          4.  Append a new name-value pair to |list| whose name is 
-              |credential|'s {{PasswordCredential/passwordName}} and whose value
-              is |credential|'s {{[[password]]}}.
-
-          5.  Push the result of running the <a>urlencoded byte serializer</a> with |list|, to
-              |body|'s <a for="body">stream</a>.
-
-          6.  Set |Content-Type| to
-              `application/x-www-form-urlencoded;charset=UTF-8`.
-
-  4.  Return |body| and |Content-Type|. 
-
   ## Security Considerations ## {#passwords-security}
 
-  ### Cross-origin Credential Leakage ### {#passwords-cross-origin-leakage}
+  ### Credential Leakage ### {#passwords-credential-leakage}
```

Comment:
I don't think the PSL bits apply to non-password credentials, do they?

I agree that the CSP bits could be merged into the broader section. In fact, most of them are there already.